### ISSUE TITLE
Change participation requirements for editors

### ISSUE THREAD

**Issue by cwadamsoforacle** [w3c/process#948]
The W3C process document (https://www.w3.org/policies/process/#publication) contains the following clause:  "An editor MUST be a participant, per [§ 3.4.2 Participation in Chartered Groups](https://www.w3.org/policies/process/#group-participation) in the Group responsible for the document(s) they are editing."

I request that this be changed from 'must' to 'should'.  The Accessibility Guidelines Working Group has found that the current language does not give us flexibility to maintain an accurate editors list.  Some editors may have contributed significantly before leaving, and subsequent publications are not significantly changed to warrant the removal of an editor.  This change will give the chairs flexibility in maintaining the list of editors.

@iadawn
@alastc

**Comment by frivoal** [comment]
I think there are two questions here:
* who is listed as "Editor" in the document, either in recognition of their active role as such, or as credit for their previous contributions in such a role
* who, as Editor, has the responsibility to "ensure that the decisions of the Group are correctly reflected in subsequent drafts of the technical report."

In my understanding, the Process's requirement is only about the second consideration, while I believe you are (at least in part) concerned about the first one.

About the second one, I think it would be quite unexpected and inappropriate for a Working Group to rely on someone who is not a participant to update its drafts. So I believe the Process is right to insist on "must be a participant".

About the first one, I'd say this is regulated by [pubrules](https://www.w3.org/pubrules/), rather than the Process itself. I don't know what requirements it has on that topic, but if they don't allow what you want to do, you should take it up with the Team, as they are the ones maintaining those rules.

I don't know if this will help you, but I will note that quite a few groups / specification have also adopted the practice of listing people who were editors but no longer are as "Former Editors" in the document, rather than "Editors".

**Comment by frivoal** [comment]
@plehegar @fantasai , would you agree with what I said above?

**Comment by iadawn** [comment]
@frivoal I am happy to raise this Team-side although one thing I note is that you have created two definitions for 'Editor' with two potentially different approaches. However, in terms of Process and PubRules, this distinction isn't there.

**Comment by frivoal** [comment]
> you have created

I do not believe _I_ have created anything, I'm just reading the Process, and that part isn't one I wrote. The Process defines an editor as the person whose responsibility it is to "ensure that the decisions of the Group are correctly reflected in subsequent drafts of the technical report". With that definition, I think it would be unreasonable to expect anybody who is not a group participant to be an editor. For instance, the Patent Policy implications of letting someone who isn't a Group participant be the one to write substantial changes to a specification are not great. More generally, it is difficult to see how someone who isn't part of the Group could feel bound by group decisions.

Whether pubrules wants people listed as Editors in the document to be editors at the time of publication, or to have been editors at some point, is not up to me, the Process CG, or the Process. I am not saying that pubrules should allow that distinction, merely that they could.

**Comment by alastc** [comment]
I don't think the scenario we are in really matches the intent of the process.

WCAG 2.1 was worked on from around 2010 to 2018, with several editors. WCAG 2.1 is now in maintence-mode, with non-substantive errata being created. Most of the work is on the associated informative documentation, but the group has approved few small errata.

The changes are less than 1% of the content, yet we have to remove the people who were responsible for the 99%.

The effect is likely to be that someone will object to the re-publication due to the change of listed editors and we will be blocked from incorporating errata into the main document.

**Comment by TallTed** [comment]
@alastc — "Former Editors" seems tailor made for your concern. Their names are not *removed from* the document; they are *moved within* the document.

**Comment by chaals** [comment]
I don't think you need to change anything. The fact that you list the 
lifetime editors isn't a real issue here, so long as the current active 
editors are participating sufficiently to follow what's happening.

How you choose to list people who contribute is up to the group - different 
groups have wildly different approaches, and I don't think that the 
possible benefits are worth the cost of trying to harmonise them.

**Comment by alastc** [comment]
Regarding:
> How you choose to list people who contribute is up to the group

and

> the Process says nothing about how you give or do not give credit to people who worked on the document.

It isn't in this case though, we cannot (according to the newly enforced respec rule) continue to list Andrew & Michael as editors. 

We could list them as "former editors", but that does not convey the reality of the situation or the work put in. As someone else put it: If someone wrote a book, would you take their name off the second edition?

**Comment by swickr** [comment]
It is important (socially, culturally, for archival and research purposes...) to acknowledge the individuals who have made significant contributions to the work. I understand this to be Florian's first point. This has career impact.

It is also important to identify who has what responsibility for each version of a document.  Our specifications are not static.  The responsibility for each version might rest with individuals, task forces, even whole groups.  The "current" version should inform readers who most recently had this responsibility.  (That might be different from who currently has the responsibility).  I understand this to be Florian's second point.

Both are vital.  Using a single term for both these, and second-order details (e.g. Working Group participants whose engagement led to the words that the Working Group adopted but who warrant distinguishing from "editor" or "author") might not be appropriate.

**Comment by rigow** [comment]
Hi all, 

I agree with the assessment that we have to distinguish between the historical list of editors and the currently active editor of a specification. 

I believe it would be foolish to require that an editor of a specification MUST or SHOULD remain in the Working Group until the document reaches recommendation or as long as a Document is worked on in the Working Group. Consequently, the list of (past & present) editors on the Specification is IMHO absolutely independent from current active membership in the Working Group. A person may have edited the document over some time and leave the group, because they change employer or for some other reason. 

The very reason that the editor MUST be a member of the group is that a variety of commitments and acknowledgements are given when an organization or person joins the Group per [§ 3.4.2 Participation in Chartered Groups](https://www.w3.org/policies/process/#group-participation). The legal mechanics that make Working Groups work in an environment that is relatively safe, legally, are all tied to Working Group membership. As an editor has a huge influence on what is written into the Specification, it is absolutely crucial that the CURRENT/ACTIVE editor is a member of the Working Group and has given those commitments. This is also reflected in [§ 6.2.6. License Grants from Non-Participants](https://www.w3.org/policies/process/#contributor-license) that goes into the same direction for patents. The copyright is not documented anymore, but should be. 

Concluding, my answer to the initial questions would be that: 

1/ Active editors MUST be members of the WG that is responsible for the Specification they are editing. The link from the [publication section](https://www.w3.org/policies/process/#publication) to [§ 3.4.2 Participation in Chartered Groups](https://www.w3.org/policies/process/#group-participation) reflects necessary legal requirements. 

2/ List of past editors are just a list and are not affected by the requirements set forth in the  [publication section](https://www.w3.org/policies/process/#publication) of the Process document. So maintaining a list of past editors is not an issue. 

Note that this is a preliminary statement (to inform this discussion)  that has not yet been coordinated with team-legal. It thus just reflects my personal opinion. If you want an authoritative answer, let me know and team-legal will provide a more formal response.

**Comment by nigelmegitt** [comment]
Observation from reading this thread: There are two categories of Editor:

1. The **current active Editor(s)** who are involved in preparing and editing a technical report that is yet to be published, e.g. for the next maturity stage, amendments etc. These are the ones the Process talks about.
2. The **historical Editor(s)** who were involved in preparing a published technical report.

Of course any individual can be in both of these categories at the same time, for a given document on which there is ongoing work.

Since the _current active Editor(s)_ for a specification might change, but the _historical Editor(s)_ are known and fixed at the time of publication, perhaps our tooling should allow for the _current active Editor(s)_ to _not_ be a part of the document, but something held elsewhere and, on change, the document is automatically updated, without considering that to be a republication, whereas the _historical Editor(s)_ are listed within the document and are considered to be a form of "acknowledgement".

**Comment by csarven** [comment]
This is less about the process and more about a convention: in the Solid Protocol, we ack'd both current and former editors and list the versions of the specification to which they contributed, e.g., "(v0.9, v0.10)".

https://solidproject.org/ED/protocol#document-editors
https://solidproject.org/ED/protocol#document-former-editors

**Comment by alastc** [comment]
Just on this point:
> It is also important to identify who has what responsibility for each version of a document. Our specifications are not static.

WCAG versions are not dynamic. As they have been adopted by regulators we can't take a living-standard approach, so the updates are very minor, literally errata. That's why it seems so unfair to bump off the people most responsible.

**Comment by TallTed** [comment]
> "former editors" ... does not convey the reality of the situation or the work put in

You can also include a freeform text acknowledgements section, usually (in my experience) as an appendix, where you can say something like, "The current Editors want to recognize the contributions of the Former Editors: Jo Smythe put in 10,000 hours of editorial effort; Mo Smythe put in 1,000 hours of editorial effort; and Bo Smythe put in 100 hours of editorial effort."

**Comment by fantasai** [comment]
I agree with @csarven and @frivoal that this is not a Process issue, but a publication conventions issue...

I suspect that the concerns about acknowledgement would be somewhat alleviated if we had "Current Editors" or "Active Editors" in parallel with "Former Editors", rather than "Editors" and "Former Editors". Or alternatively take Florian's suggestion to make "Editors" a statement unbounded by time, encompassing all editors past and previous--but this has the disadvantage of making it unclear who is currently responsible.

It's probably a good idea to align on some kind of acceptable common convention, so while I think it's out-of-scope for the Process CG, I'll leave the issue open for discussion. Or maybe @plehegar can transfer it to the correct place.

**Comment by plehegar** [comment]
I'm happy to transfer the issue next week, unless something comes up.

I concur that we need to fix this issue in the Guide.

There are a few dimensions to consider:
- what are the roles? (eg [XMLSignature](https://www.w3.org/Signature/Contributor.html))
- where do they appear in the document?
- how does those roles get translated into tooling, such as specref.org ?

**Comment by michael-n-cooper** [comment]
I was just made aware of this GH issue, and see that some points I made elsewhere are being imperfectly re-made here without acknowledging the source. I am copying, with adaptations, some comments I made in an email discussion that led to the filing of this issue. Note that my comments are specifically about a publication where the version number of the document is not changed, not about new versions of specs.

To begin with, it really surprises me that a publication, which does not require a version number change, would require a change of editors. If a second edition of a book is printed, which fixes typos from the first edition, the author of the book is not changed to be the person who fixed the typos.

The meaning of "editor" of a W3C spec has been historically loosely defined, but generally reflected significant leadership contribution to the spec. The use of "authors" was introduced in the 2010s, but WCAG 2.0 was published before that convention. All the editors of that document, myself included, are also "authors" from the meaningful technical, content, and process leadership exercised. This situation is also true for WCAG 2.1, where Andrew's leadership was singular in making that version happen.

Moving people to a "former editors" list has an effect of downgrading the perception of their contribution, notwithstanding any date clarifications in that section. My work on WCAG 2.0 is an important part of my career portfolio, and if that were republished with my name moved to the former editors section, links I provide about my background would suddenly go to a page that has changed in a way that could impact my professional perception. In the case of WCAG 2.1, republishing it with a change of editors would massively distort peoples' perception of the document's provenance, as an editor who came in at the end would seem to have replaced the editor who made the document happen. 

I don't disagree with the use of a former editors list. I only disagree with moving people who are listed as editors of a Recommendation to that list in updates of the same version of that Recommendation, for these credit and provenance reasons.

**Comment by frivoal** [comment]
(with the repeated caveat that this is not a Process discussion, since the Process isn't the one mandating anything about how we give credit to editors, whether currently active or not…)

I agree this is important, and that giving credit matters. I think the challenge we're facing is (at least in part) because the Editor's list in the document serves multiple purposes, and that these purposes are in tension:
* give credit where credit is due. Not served well when people are left off or removed
* enabling readers and (would be) contributors to identify the people who are currently responsible for further evolution of the document, in case they want to get in touch. Not served well when (a) we also list people who made significant contributions but are no longer active, and (b) someone new has taken on that role since we last published, but won't be reflected until we re-publish

**Comment by frivoal** [comment]
Thanks @deniak, I believe this addresses the issue as stated. In that case, I suggest we don't need to transfer it, and my simply close it.
